project/script_TP.tcl

```diff
@@ -11,24 +11,25 @@ open_project -reset trackletProcessor
 
 # source files
 set CFLAGS {-std=c++11 -I../TrackletAlgorithm}
-set_top TrackletProcessor_L1L2D
+set_top TrackletProcessor_L3L4C
 add_files ../TrackletAlgorithm/TrackletProcessor.cc -cflags "$CFLAGS"
-add_files -tb ../TestBenches/TrackletProcessor_L1L2D_test.cpp -cflags "$CFLAGS"
+add_files -tb ../TestBenches/TrackletProcessor_L3L4C_test.cpp -cflags "$CFLAGS"
 
 open_solution "solution1"
 
 # Define FPGA, clock frequency & common HLS settings.
 source settings_hls.tcl
+set_clock_uncertainty .05
 
 # data files
 add_files -tb ../emData/TP/tables/
-add_files -tb ../emData/TP/TP_L1L2D/
+add_files -tb ../emData/TP/TP_L3L4C/
 
 csim_design -mflags "-j8"
 csynth_design
 cosim_design 
-export_design -format ip_catalog
+#export_design -format ip_catalog
 # Adding "-flow impl" runs full Vivado implementation, providing accurate resource use numbers (very slow).
-#export_design -format ip_catalog -flow impl
+export_design -format ip_catalog -flow impl
```

Comment:
@jasonfan393 Please switch the comments on these lines so that the implementation is _not_ run by default. It's useful for us because it gives a more accurate timing estimate, but this Tcl file is used by the CI tests, which we don't want to take too long to run.